src/test/java/org/opensrp/web/rest/CustomErrorResourceTest.java

```diff
@@ -0,0 +1,51 @@
+package org.opensrp.web.rest;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+import org.opensrp.web.GlobalExceptionHandler;
+import org.opensrp.web.rest.it.TestWebContextLoader;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import static org.junit.Assert.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(loader = TestWebContextLoader.class, locations = { "classpath:test-webmvc-config.xml" })
+public class CustomErrorResourceTest {
+
+	private MockMvc mockMvc;
+
+	private String ERROR_ENDPOINT = "/error";
+
+	@InjectMocks
+	private CustomErrorResource customErrorResource;
+
+	@Before
+	public void setUp() {
+		MockitoAnnotations.initMocks(this);
+
+		mockMvc = MockMvcBuilders.standaloneSetup(customErrorResource)
+				.setControllerAdvice(new GlobalExceptionHandler())
+				.build();
```

Comment:
Init the whole context and test make a request that would be caught by CustomErrorResource